components/Cards/src/CardThemer/MDCCardThemer.m

```diff
@@ -29,7 +29,7 @@ + (void)applyScheme:(nonnull id<MDCCardScheming>)scheme
              toCard:(nonnull MDCCard *)card {
   [card setShadowElevation:kNormalElevation forState:UIControlStateNormal];
   [card setShadowElevation:kHighlightedElevation forState:UIControlStateHighlighted];
-
+  [card setInteractable:YES];
```

Comment:
super-nit: Prefer properties. (Except in init methods)
card.interactable = YES
The Modernize Objective-C tool in Xcode will point these instances out.